packages/legend-studio/src/components/editor/side-bar/LocalChanges.tsx

```diff
@@ -215,45 +239,99 @@ export const LocalChanges = observer(() => {
           >
             <GoSync />
           </button>
+          <button
```

Comment:
Let's not expose this. We will unify all of this into `sync workspace` action, now the sync icon will make sense.

Reply:
I think this is valid. We should have a way to sync the workspace to the latest revision if outdated.

Reply:
Will resolve this thread, I back out from my push for `unified syncing behavior`